packages/fluentui/react-bindings/src/context-selector/useContextSelectors.ts

```diff
@@ -46,7 +46,7 @@ export const useContextSelectors = <
 
       if (payload[0] <= version) {
         const stateHasNotChanged = Object.keys(selectors).every((key: Properties) =>
-          Object.is(prevState[1][key] as SelectedValue, selected[key]),
+          Object.is((prevState[1] as { [key: string]: any })[key] as SelectedValue, selected[key]),
```

Comment:
I managed to build correctly without this change, is it needed ?

Reply:
I reverted changes in this file, tried locally and it breaks without them. Let's leave if for now if you don't have better suggestions.